corehq/apps/groups/models.py

```diff
@@ -222,7 +235,7 @@ def get_reporting_groups(cls, domain):
             startkey=key,
             endkey=key + [{}],
             include_docs=True,
-            # stale=settings.COUCH_STALE_QUERY,
+            stale=settings.COUCH_STALE_QUERY,
```

Comment:
The reason why I wasn't concerned is because ```groups/by_name``` is called as stale in the dbaccessors.  That said I could be convinced to leave as is pretty easily